### PR TITLE
Add RU an EN Zevaykin PB

### PR DESCRIPTION
**Pull request by Pseudolukian**
### Changelog category
* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/en/core/public-materials/_includes/conferences/2024/Qcon.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### {{ ydb-short-name }}: dealing with Big Data and moving towards AI {#2024-qcon}
```
Comment (blinkov) [inline ydb/docs/en/core/public-materials/_includes/conferences/2024/Qcon.md:1]: @Pseudolukian, this file is not included anywhere and thus isn't rendered.

File: ydb/docs/en/core/public-materials/_includes/conferences/2024/Qcon.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The database provides true elastic scalability, capable of scaling up or down by several orders of magnitude.
+
+At the same time, the database is fault-tolerant. It has been designed to work across three availability zones, ensuring that the database continues to function even if one of the zones becomes unavailable. The database automatically recovers after a disk failure, server failure, or datacenter failure, with minimal latency disruptions to the applications.
+
+Currently, work is underway to implement accurate and approximate nearest neighbor search for machine learning purposes.
+
+**Takeaways**:
+* Architecture of a distributed fault-tolerant database.
+* Approaches to the implementation of vector search on large amounts of data.
```
Comment (blinkov) [inline ydb/docs/en/core/public-materials/_includes/conferences/2024/Qcon.md:19]:
```suggestion
{{ ydb-short-name }} is a versatile, open-source Distributed SQL database management system that combines high availability and scalability with strong consistency and ACID transactions. It provides services for machine learning products and goes beyond traditional vector search capabilities.

**Outline**:

This database is used for industrial operations within Yandex. Among its clients are Yandex Market, Yandex Alice, and Yandex Taxi, which are some of the largest and most demanding AI-based applications.

The database offers true elastic scalability, capable of scaling up or down by several orders of magnitude.

Simultaneously, the database is fault-tolerant. It is designed to operate across three availability zones, ensuring continuous operation even if one of the zones becomes unavailable. The database automatically recovers from disk failures, server failures, or data center failures, with minimal latency disruptions to applications.

Currently, work is underway to implement accurate and approximate nearest neighbor searches for machine learning purposes.

**Takeaways**:
* Architecture of a distributed, fault-tolerant database.
* Approaches to implementing vector search on large datasets.
```

File: ydb/docs/ru/core/public-materials/_includes/conferences/2024/Saint_HighLoad.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+### Гарантии доставки сообщений в {{ ydb-short-name }} Topics {#2024-saint-highload}
+
+{% include notitle [database_internals_tag](../../tags.md#database_internals) %}
+
+[{{ team.zevaykin.name }}]({{ team.zevaykin.profile }}) ({{ team.zevaykin.position }}) рассмотрел базовые проблемы ненадёжной передачи данных и способы борьбы с ними: повторы и дедупликация. Александр на примере паттернов микропроцессорной архитектуры проиллюстрировал гарантии доставки: at-most-once, at-least-once, exactly-once.
+
+@[YouTube](https://youtu.be/6l64n8t8Ivs?si=coC70xmfuaoIzxPA)
+
+В заключение Александр на примере двух брокеров очередей сообщений (Kafka и {{ ydb-short-name }} Topics) показывает детали реализации гарантий доставки.
```
Comment (blinkov) [inline ydb/docs/ru/core/public-materials/_includes/conferences/2024/Saint_HighLoad.md:9]:
```suggestion
[{{ team.zevaykin.name }}]({{ team.zevaykin.profile }}) ({{ team.zevaykin.position }}) рассмотрел основные проблемы ненадёжной передачи данных и методы борьбы с ними: повторы и дедупликацию. Александр на примере паттернов микропроцессорной архитектуры продемонстрировал гарантии доставки: at-most-once, at-least-once, exactly-once.

@[YouTube](https://youtu.be/6l64n8t8Ivs?si=coC70xmfuaoIzxPA)

В заключение Александр на примере двух брокеров очередей сообщений — Kafka и {{ ydb-short-name }} Topics — показал детали реализации гарантий доставки.
```

File: ydb/docs/ru/core/public-materials/_includes/conferences/2024/Yandex_Infrastructure.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+### {{ ydb-short-name }} Topics: история взаимоотношений с Kafka {#2024-ya-inf}
+
+{% include notitle [database_internals_tag](../../tags.md#database_internals) %}
+
+Шина данных уровня Яндекса требует нестандартных решений. [{{ team.zevaykin.name }}]({{ team.zevaykin.profile }}) ({{ team.zevaykin.position }}) рассказал о том, как появился сервис {{ ydb-short-name }} Topics, и что такое применённые оптимизации. Также Александр рассказал об архитектуре {{ ydb-short-name }} Topics и раскрыл причины отказа от Kafka. 
+
+@[YouTube](https://youtu.be/lZ5SUzdQO7k?si=cWx3sE6eDE_sx7f8)
+
+В заключение, вы узнаете как команда {{ ydb-short-name }} сделала Kafka API в режиме serverless.
```
Comment (blinkov) [inline ydb/docs/ru/core/public-materials/_includes/conferences/2024/Yandex_Infrastructure.md:9]:
```suggestion
Шина данных уровня Яндекса требует нестандартных решений. [{{ team.zevaykin.name }}]({{ team.zevaykin.profile }}) ({{ team.zevaykin.position }}) рассказал о том, как появился сервис {{ ydb-short-name }} Topics и какие оптимизации были в нём применены. Также Александр описал архитектуру {{ ydb-short-name }} Topics и объяснил, почему было принято решение отказаться от Kafka.

@[YouTube](https://youtu.be/lZ5SUzdQO7k?si=cWx3sE6eDE_sx7f8)

В завершение вы узнаете, как команда {{ ydb-short-name }} реализовала Kafka API в режиме serverless.
```

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/public-materials/videos.md [hunk 5]
```diff
@@ -9,6 +9,10 @@ Video recordings from conferences and webinars. The materials are divided by cat
 
 - 2024
 
+  {% include [Qcon](./_includes/conferences/2024/Qcon.md) %}
+
+  {% include [stackconf](./_includes/conferences/2024/stackconf.md) %}
+
   {% include [KubeCon + CloudNativeCon + Open Source Summit](./_includes/conferences/2024/kubecon-hongkong.md) %}
 
   {% include [COSCUP](./_includes/conferences/2024/coscup.md) %}
```
Comment (blinkov) [inline ydb/docs/en/core/public-materials/videos.md:18]:
```suggestion
  {% include [KubeCon + CloudNativeCon + Open Source Summit](./_includes/conferences/2024/kubecon-hongkong.md) %}

  {% include [COSCUP](./_includes/conferences/2024/coscup.md) %}
  
  {% include [Qcon](./_includes/conferences/2024/Qcon.md) %}

  {% include [stackconf](./_includes/conferences/2024/stackconf.md) %}
```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**